packages/mgt-element/src/IGraph.ts

```diff
@@ -66,3 +66,24 @@ export interface IGraph {
    */
   createBatch(): IBatch;
 }
+
+export type GraphEndpoint =
```

Comment:
We should have documentation comments on all exported types, objects and public methods